test/simulator/main/org/apache/cassandra/simulator/SimulatorUtils.java

```diff
@@ -40,9 +40,10 @@ public static RuntimeException failWithOOM()
     public static void dumpStackTraces(Logger logger)
     {
         Map<Thread, StackTraceElement[]> threadMap = Thread.getAllStackTraces();
-        threadMap.forEach((thread, ste) -> {
-            logger.error("{}:\n   {}", thread, Threads.prettyPrint(ste, false, "   ", "\n", ""));
-        });
+        String prefix = "   ";
+        String delimiter = "\n" + prefix;
```

Comment:
`delimiter` doesn't include the `prefix` so to make the logs clean, I had to manually add